modules/RTC/RTCUtils.js

```diff
@@ -1181,6 +1183,31 @@ var RTCUtils = {
      */
     getAudioOutputDevice: function () {
         return audioOutputDeviceId;
+    },
+
+    /**
+     * Returns list of available media devices if its obtained, otherwise an
+     * empty array is returned/
+     * @returns {Array} list of available media devices.
+     */
+    getCurrentlyAvailableMediaDevices: function () {
+        return currentlyAvailableMediaDevices;
+    },
+
+    /**
+     * Returns event data for device to be reported to stats.
+     * @returns {MediaDeviceInfo} device.
+     */
+    getEventDataForActiveDevice: function (device) {
+        var devices = [];
+        var deviceData = {
```

Comment:
There is internal function mediaDeviceInfoToJSON here https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/RTC/RTCUtils.js#L307
I think we can expose it and re-use. WDYT?